eks-hybrid-iam-ra-setup.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -eux -o pipefail
+
+declare -r SECRETS_DIR="/.bottlerocket/rootfs/root/.aws"
+DRY_RUN="false"
+
+set +x
+for opt in "$@"; do
+    optarg="$(expr "${opt}" : '[^ =]*[= ]\(.*\)')"
+    case "${opt}" in
+       --certificate=*) NODE_CERT_DATA="${optarg}" ;;
+       --key=*) NODE_KEY_DATA="${optarg}" ;;
+       --dry-run=*) DRY_RUN="${optarg}" ;;
+    esac
+done
```

Comment:
Can you fix the spacing across this file to be consistent? Here it's 3 spaces for indentation, elsewhere it's 4.